ci/all_osp13.yml

```diff
@@ -31,29 +31,55 @@ alias:
 scale:
 # lab specific vars
   lab_url: "http://quads.rdu2.scalelab.redhat.com"
-  machine_types:
-    supermicro:
-      1029p: 
-        interfaces: [enp94s0f0, enp94s0f1, enp94s0f2, enp94s0f3]
-      1029u:
-        interfaces: [enp175s0f0, enp175s0f1, enp216s0f0, enp216s0f1]
-      6048r:
-        interfaces: [enp4s0f0, enp4s0f1, enp131s0f0, enp131s0f1]
-      5039ms:
-        interfaces:  [enp1s0f0, enp1s0f1, enp2s0f1]
-      6049p:
-        interfaces: [enp175s0f0, enp175s0f1, enp216s0f0, enp216s0f1]
-      6048p:
-        interfaces:  []
-    dell:
-      r620: 
-        interfaces: [enp66s0f2, enp66s0f3, enp1s0f0, enp1s0f1]
-      r630: 
-        interfaces: [enp1s0f0, enp1s0f1, enp1s0f2, enp1s0f3]
-      r730xd:
-        interfaces: [enp1s0f0, enp1s0f1, enp130s0f0, enp130s0f1]
-      r930:
-        interfaces: [enp1s0f0, enp1s0f1, enp10s0f0, enp10s0f1]
+  rhel7:
+    machine_types:
+      supermicro:
+        1029p:
```

Comment:
Can you please organise these like below (please remove interfaces key and in ansible tasks also)
http://paste.openstack.org/show/792800/